keep_joints:
  - x_taxel_01_uSPa35_01_joint
  - x_taxel_01_uSPa35_02_joint
  - x_taxel_01_uSPa35_03_joint
  - x_taxel_01_uSPa35_04_joint
  - x_taxel_01_uSPa35_05_joint
  - x_taxel_01_uSPa35_06_joint
  - x_taxel_01_uSPa35_07_joint
  - x_taxel_01_uSPa35_08_joint
  - x_taxel_01_uSPa35_09_joint
  - x_taxel_01_uSPa35_10_joint
  - x_taxel_01_uSPa35_11_joint
  - x_taxel_01_uSPa35_12_joint
  - x_taxel_01_uSPa35_13_joint
  - x_taxel_01_uSPa35_14_joint
  - x_taxel_01_uSPa35_15_joint
  - x_taxel_02_uSPa35_01_joint
  - x_taxel_02_uSPa35_02_joint
  - x_taxel_02_uSPa35_03_joint
  - x_taxel_02_uSPa35_04_joint
  - x_taxel_02_uSPa35_05_joint
  - x_taxel_02_uSPa35_06_joint
  - x_taxel_02_uSPa35_07_joint
  - x_taxel_02_uSPa35_08_joint
  - x_taxel_02_uSPa35_09_joint
  - x_taxel_02_uSPa35_10_joint
  - x_taxel_02_uSPa35_11_joint
  - x_taxel_02_uSPa35_12_joint
  - x_taxel_02_uSPa35_13_joint
  - x_taxel_02_uSPa35_14_joint
  - x_taxel_02_uSPa35_15_joint
